src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -449,6 +451,19 @@ public async Task<ReviewRevisionModel> CreateMasterReviewAsync(ClaimsPrincipal u
                 }
             }
             await _reviewsRepository.UpsertReviewAsync(review);
+
+            //Update comments to relink them to last revision if it's linked to deleted revsions. 
+            if (deletedRevisionIds.Any())
+            {
+                var comments = await _commentsRepository.GetCommentsAsync(review.ReviewId);
+                var lastRevisionId = review.Revisions.Last().RevisionId;
+                comments = comments.Where(c => deletedRevisionIds.Contains(c.RevisionId));
+                foreach (var comment in comments)
+                {
+                    comment.RevisionId = lastRevisionId;

```

Comment:
What if the member is not there anymore?

Reply:
One solution I was thinking of is to show them as outdated comments in conversation. But we will not have any context to show since we don't save any context currently with each comment.

Reply:
done. Auto delete will not remove revisions with a comment